README.md

```diff
@@ -23,7 +23,7 @@ For the full instructions on how to setup your environment for React Native deve
 1. Run the project setup script:
 
 ```bash
-yarn setup
+./scripts/setup
 ```
```

Comment:
with yarn berry, scripts can't be executed before the yarn cache is built. so you have to first `yarn` then run scripts. running the setup directly will do all of that for you automatically.